server/controllers/registration-controller.js

```diff
@@ -25,7 +25,7 @@ const registration = (req, res) => {
         delete postData.phone_number;
         postData.method = "";
       }
-      if (settings && settings.subscriptions && postData.billing_info) {
+      if (settings && settings.subscriptions) {
```

Comment:
there are only two methods to register SMS and credit/debit card. The else clause above rules out that the registration method is SMS. Only other possibility is credit/debit card, hence this condition check should be sufficient.

Reply:
I don't think this is correct, it's also possible to disable SMS verification and just register with email.
I believe it's necessary to add an extra parameter that tells the backend with 100% certainty this is a registration which requires a credit/debit card payment.